src/main/java/org/apache/accumulo/examples/mapreduce/bulk/BulkIngestExample.java

```diff
@@ -121,9 +120,15 @@ public static void main(String[] args) throws Exception {
       try (PrintStream out = new PrintStream(
           new BufferedOutputStream(fs.create(new Path(workDir + "/splits.txt"))))) {
         Collection<Text> splits = client.tableOperations().listSplits(SetupTable.tableName, 100);
-        for (Text split : splits)
-          out.println(Base64.getEncoder().encodeToString(TextUtil.getBytes(split)));
+        for (Text split : splits) {
+          byte[] bytes = split.getBytes();
+          if (bytes.length != split.getLength()) {
+            out.println(Base64.getEncoder().encodeToString(split.copyBytes()));
```

Comment:
I think your logic is a little wrong. If the lengths do not match, `out.println` is called twice.  I would use the logic below:
```
        for (Text split : splits)
          out.println(Base64.getEncoder().encodeToString(split.copyBytes()));
```
The copyBytes() method is not as efficient but this is just test code.

Reply:
I actually had that at one point because i figured the same thing but I assumed it would be best to match the code as much as possible as the previous. The textUtil getBytes() function had this form but I do realize now that you're right in that it prints twice.

Reply:
I could also put the second print line in the else portion if that would be preferred.

Reply:
No worries. Just push another commit that fixes it and I will merge.  Thanks for the PR!

Reply:
> I could also put the second print line in the else portion if that would be preferred.

This is example code. I think only using copyBytes() is easier to understand.  TextUtil.getBytes() was only created for efficiency (which is not needed here).  However, I am OK with a if-else too.

Reply:
Even if this was not example this code does not need to be efficient because its called once during map reduce setup.  The code is not calling for each key/value in the map reduce job.

Reply:
@mikewalch I pushed the change. Sorry for the delay, I was out of the office early yesterday.